src/server.ts

```diff
@@ -86,16 +89,72 @@ app.get("/data/:chainId/rounds/:roundId/applications.csv", async (req, res) => {
       application.metadata.application.project.projectTwitter,
       application.metadata.application.project.projectGithub,
       application.metadata.application.project.userGithub,
-      ...answers,
+      ...answers
     ]);
   }
 
   res.setHeader("content-type", "text/csv");
   res.send(csv.getHeaderString() + csv.stringifyRecords(records));
 });
 
+app.get("/data/:chainId/rounds/:roundId/votecoefficients.csv", async (req, res) => {
+  const db = loadDatabase(req.params.chainId);
+
+  const votes = await db
+    .collection(`rounds/${req.params.roundId}/votes`)
+    .all();
+
+  const data = fs.readFileSync("./data/passport_scores.json", {
+    encoding: "utf8",
+    flag: "r"
+  });
+
+  /*TODO: type this once we have unified types*/
+  const passportScores: any[] = JSON.parse(data);
+
+  const voteCoefficients = votes.map((vote) => {
+    const {
+      address,
+      evidence,
+      error,
+      ...passport_score
+    } = passportScores.find(score => score.address.toLowerCase() === vote.voter.toLowerCase()) ?? {};
+    return {
+      ...vote,
+      ...passport_score,
+      ...evidence
+    };
+  });
+
+  const records = voteCoefficients.map((voteCoefficient) => Object.values(voteCoefficient));
```

Comment:
To add to this, keep in mind that the votes file could potentially have lots of votes, we should probably do the join with passport and convert to CSV record in one pass, instead of walking a big array twice.

Reply:
Good catch, I was thinking this might be an issue. Will fix.

Reply:
fixed in latest push

Reply:
we're still looping through the array twice, once to join with passport and the second time to convert it to a CSV row, we should do this in one pass